deployment/windows/jellyfin.nsi

```diff
@@ -0,0 +1,252 @@
+; Shows a lot of debug information while compiling
+; This can be removed once stable.
+!verbose 4
+;--------------------------------
+
+    !include "MUI2.nsh"
+	
+    Var _JELLYFINVERSION_
+    Var _JELLYFINDATADIR_
+    Var _SERVICEINSTALLED_
+	Var _EXISTINGINSTALLATION_
+	Var _EXISTINGSERVICE_
+
+;--------------------------------
+;General
+
+; Align installer version with jellyfin.dll version
+    !getdllversion "$%InstallLocation%\jellyfin.dll" ver_
+    !echo "jellyfin.dll version is ${ver_1}.${ver_2}.${ver_3}.${ver_4}" ;!echo will print it while building
+
+    Name "Jellyfin ${ver_1}.${ver_2}.${ver_3}.${ver_4}"
+    OutFile "Jellyfin.Installer.${ver_1}.${ver_2}.${ver_3}.${ver_4}.exe"
+    BrandingText "Jellyfin ${ver_1}.${ver_2}.${ver_3}.${ver_4} Installer"
+
+; installer attributes
+    VIProductVersion "${ver_1}.${ver_2}.${ver_3}.${ver_4}"
+    VIFileVersion "${ver_1}.${ver_2}.${ver_3}.${ver_4}"
+    VIAddVersionKey "ProductName" "Jellyfin"
+    VIAddVersionKey "FileVersion" "${ver_1}.${ver_2}.${ver_3}.${ver_4}"
+	VIAddVersionKey "LegalCopyright" "Jellyfin, Free Software Media System"
+	VIAddVersionKey "FileDescription" "Jellyfin, Free Software Media System"  
+	
+;Default installation folder
+    InstallDir "$APPDATA\Jellyfin"
+  
+;Get installation folder from registry if available
+    InstallDirRegKey HKLM "Software\Jellyfin" "InstallLocation"
```

Comment:
If we are storing this in `HKLM` then the install location should be `Program Files` too (system wide).